js/scripts.js

```diff
@@ -0,0 +1,107 @@
+    },
+  },
+  {
+    id: 4,
+    img: './assets/Home/speaker-04.svg',
+    featuredetail: {
+      name: 'Kilnam Chon',
+      positon: '',
+      description: 'Kilnam Chon helped bring the internet to Asia and is an outspoken advocate for the open web and digital com-mons. In 2012. he was inducted into the inaugural class of the Internet Society’s (ISOC) Internet Hall of Fame',
+    },
+  },
+  {
+    id: 5,
+    img: './assets/Home/speaker-05.svg',
+    featuredetail: {
+      name: 'Julia Leda',
+      positon: 'President of Young Pirates of Europe',
+      description: 'European ingetration, political democracy and participation of youth through online as her major condern, Reda’s report outlining potential changes to EU copyright law was approved by the Parliament in July',
+    },
+  },
+  {
+    id: 6,
+    img: './assets/Home/speaker-06.svg',
+    featuredetail: {
+      name: 'Ryan Merkley',
+      positon: 'CEO of Creativve Commons, ex COO of the Mozilla Foundation',
+      description: 'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+    },
+  },
+];
+
+let displayedFeatures = features.slice(0, 2);
+
+function showFeatures() {
+  const projectFeatures = document.querySelector('.featured');
+  projectFeatures.innerHTML = '<h2>Featured Speakers<br> <img src="./assets/Home/feature-title-underline.svg" alt="feature underline"></h2>';
+  let wholeFeatures = '';
+  const screenSize = window.matchMedia('(min-width: 768px)');
+  if (screenSize.matches) {
+    displayedFeatures = features.slice();
+  }
+  for (let i = 0; i < displayedFeatures.length; i++) {
+    wholeFeatures += `
+    <div class="featureds hide">          
+    <ul class="spkr">
+        <li class="spkr1"> <img src="${displayedFeatures[i].img}" alt="${displayedFeatures[i].alt}">
+        <ul class = "spkrdetails">
+            <li class="spkr2"><h3>${displayedFeatures[i].featuredetail.name}</h3></li>
+            <li class="spkr2"><h4>${displayedFeatures[i].featuredetail.positon}</h4></li>
+            <li class="spkr2"><hr></li>
+            <li class="spkr2"><p>${displayedFeatures[i].featuredetail.description}</p></li>
+        </ul>
+    </ul>
+</div>
+`;
+  }projectFeatures.innerHTML += `<div class ="tworows">${wholeFeatures}</div>`;
+  if (displayedFeatures.length === features.length) {
+    projectFeatures.innerHTML += '<div class="loadMore" onclick="showMore()" style=""><button>Less<img  src = "./assets/Home/arrow_up_icon.svg"></button></div>';
+  } else {
+    projectFeatures.innerHTML += '<div class="loadMore" onclick="showMore()" style=""><button>More<img  src = "./assets/Home/arrow_down_icon.svg"></button></div>';
+  }
+  projectFeatures.appendChild();
+}
+
+function popmenu() {
+  const mobileMenu = document.querySelector('.showcontent');
+  if (mobileMenu.id === 'hidden') {
+    mobileMenu.id = 'visibile';
+  } else {
+    mobileMenu.id = 'hidden';
+  }
+}
```

Comment:
- [OPTIONAL] Good job adding the menu, I think it would be better when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. In your case when I click any option for example `About` of the mobile menu, the menu doesn’t disappear and I need to close it by myself. It would be better if it's automatic 😉 😉